src/ewmh_m2m/__main__.py

```diff
@@ -36,6 +36,10 @@ def move_to_screen(args):
             raise ValueError("No sibling screen found")
 
         new_window_geometry = relative_geometry.build_absolute(new_screen)
+        if args.no_resize:
+            new_window_geometry.w = window_geometry.w
+            new_window_geometry.h = window_geometry.h
+
```

Comment:
I cannot test it for the moment. Can you elaborate a little bit on what you try to do ?
If you want to keep the initial size of the window, why not use `initial_window_geometry` instead of `window_geometry`.
Actually, I don’t remember the difference between these both variables. Do we need 2 different ones or can we use the same ?

Anyway, instead of overriding the `new_window_geometry` fields, I would recommend something like:
```suggestion
        if args.no_resize:
            win.geometry = initial_window_geometry
        else:
            new_window_geometry = relative_geometry.build_absolute(new_screen)
            _logger.debug("New window geometry: %s", new_window_geometry)
            win.geometry = new_window_geometry
```